Libplanet/Tx/Transaction.cs

```diff
@@ -584,11 +580,12 @@ internal RawTransaction ToRawTransaction(bool includeSign)
                     a.ToByteArray()).ToArray(),
                 publicKey: PublicKey.Format(false),
                 timestamp: Timestamp.ToString(TimestampFormat),
-                actions: Actions.Select(a => new Dictionary<string, object>
-                {
-                    { "type_id", ActionTypeAttribute.ValueOf(a.GetType()) },
-                    { "values", a.PlainValue },
-                })
+                actions: Actions.Select(a =>
+                    a.PlainValue.ToDictionary(
```

Comment:
Maybe it's better to change the type of `RawTransaction.Actions`... 😕

Reply:
Currently `RawTransaction` consists of many mutable types like `byte[]` and `Dictionary<K, V>`.  If we want `RawTransaction` to be freer from state I believe we need to adjust other fields too.  Seems like it should be a separated patch.